.github/workflows/check-unity-sdk-for-updates.yml

```
@@ -0,0 +1,55 @@
+name: Check Unity SDK Updates
+
+on:
```

```suggestion
on:
  push:
    branches:
      - main
```
I'd also add it on every PR merge to ensure that the dependencies are up to date asap